ops/main.py

```diff
@@ -326,7 +323,6 @@ def _should_use_controller_storage(db_path: Path, meta: ops.charm.CharmMeta) ->
     """Figure out whether we want to use controller storage or not."""
     # if you've previously used local state, carry on using that
     if db_path.exists():
-        logger.debug("Using local storage: %s already exists", db_path)
```

Comment:
I'd consider having a message here when we *do* create a new db file.

Reply:
there's four returns in this function
1 - local storage exists, all cool, nothing to report (no log)
2- we're a machine charm -> use local storage. (log)
3 - current version has controller storage -> use it. (log)
4 - none of the above -> use local storage (log) 

I thought that since we already have logging output for each of the 'less common' cases 2-3-4, it had little value to add one more logging line that basically only says 'we're not in case 1'.

Reply:
Or, pardon my density, are you saying we should be adding a debug log somewhere not in this function, at the point in which the database is actually created (which should only happen once, I guess)?

Reply:
In that case we need to manually go and check if the file exists already or not (and in that case log), because `sqlite3.connect` will silently create the file if it doesn't exist